src/Sylius/Component/Registry/spec/DependencyInjection/Compiler/AliasedServicePassSpec.php

```diff
@@ -0,0 +1,134 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace spec\Sylius\Component\Registry\DependencyInjection\Compiler;
+
+use PhpSpec\ObjectBehavior;
+use Sylius\Component\Registry\DependencyInjection\Compiler\AliasedServicePass;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Definition;
+use Sylius\Component\Registry\ServiceRegistry;
+use Symfony\Component\DependencyInjection\Reference;
+
+/**
+ * @author Daniel Leech <daniel@dantleech.com>
+ */
+class AliasedServicePassSpec extends ObjectBehavior
```

Comment:
Possibly in addition to this spec, although failure in Sylius is all or nothing if we replace all the existing service registry compiler passes - and even then chance of failure is more likely at the level of instantiating the pass (e.g. specifying the wrong tag/alias).